cmd/operator-sdk/build/cmd.go

```diff
@@ -63,6 +65,8 @@ For example:
 	buildCmd.Flags().StringVar(&testLocationBuild, "test-location", "./test/e2e", "Location of tests")
 	buildCmd.Flags().StringVar(&namespacedManBuild, "namespaced-manifest", "deploy/operator.yaml", "Path of namespaced resources manifest for tests")
 	buildCmd.Flags().StringVar(&dockerBuildArgs, "docker-build-args", "", "Extra docker build arguments as one string such as \"--build-arg https_proxy=$https_proxy\"")
+	buildCmd.Flags().StringVar(&buildahBuildArgs, "buildah-build-args", "", "Extra buildah build arguments as one string such as \"--build-arg https_proxy=$https_proxy\"")
```

Comment:
WDYT of re-using a flag `build-args` for both docker and buildah?

Reply:
Can we rename ```docker-build-args``` with something like ```image-build-args```?

Then re-using flag makes sense to me. I'm not sure how operator-sdk community thinks about the backward compatibility of command line option.

Reply:
`image-build-args` is fine with me. We'll have to add these changes to the `CHANGELOG.md` too.